ui/test/nuxeo-date-picker.test.js

```diff
@@ -32,6 +36,20 @@ function testValue(element, value, isUTC) {
   // expect(element._pickerValue).to.be.equal(local);
 }
 
+function testValueWithLocale(element, value, locale, isUTC) {
+  const currentLocale = moment.locale();
+  moment.locale(locale);
+  element.value = value;
+  expect(element.value).to.be.equal(value);
+  const inputValue = getInputDisplay(element).value;
+  let val = isUTC ? moment.utc(value) : moment(value).local();
```

Comment:
Instead of changing the locale and altering it later, which can be a problem if the test fails and the restoration is not completed, we could try to instead change the locale on the moment object we create here. Something along these lines (untested):
```suggestion
  let val = isUTC ? moment.utc(value) : moment(value).local();
  val.locale(locale);
```

Reply:
This doesn't work, but I'll look at the API to see if I can do something similar.

Reply:
@Gabez0r in fact this works, val is represented in the locale provided, but in the end the test doesn't produce the expected result. Let me clarify why. 

If you don't change the locale used by moment (setting it globally), the input inside the `nuxeo-date-picker` element is not doing to display the date in the correct locale, and we end up comparing dates in the `en` local.

Reply:
This happens because the date value is not the same as the one displayed by the widget. We can over this together if you feel it's necessary.

Reply:
Ah, a typical case were we should be using dependency injection to fix this. 😬 Well, if we need to change it globally, I think it would be safer to have a dedicated suit, and set the language on the setup and restore it on the teardown, wdyt?

Reply:
Exactly what I thought. I am going to push these changes 👍 thanks